src/test/java/uk/gov/dhsc/htbhf/BrowserStackDriverWrapper.java

```diff
@@ -0,0 +1,88 @@
+package uk.gov.dhsc.htbhf;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.Validate;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+import static uk.gov.dhsc.htbhf.BrowserStackCapabilities.buildDesiredCapabilities;
+import static uk.gov.dhsc.htbhf.BrowserStackCapabilities.getBrowserStackCapabilities;
+
+/**
+ * Controls building of the WebDriver and WebDriverWait needed for BrowserStack testing so that we can rebuild them when required.
+ */
+@Slf4j
+public class BrowserStackDriverWrapper implements WebDriverWrapper {
+
+    private static final String BROWSER_STACK_URL = "https://hub-cloud.browserstack.com/wd/hub";
+
+    private final int waitTimeoutInSeconds;
+
+    private final DesiredCapabilities desiredCapabilities;
+
+    private final URL browserStackUrl;
+
+    private WebDriver webDriver;
+
+    private WebDriverWait webDriverWait;
+
+    public BrowserStackDriverWrapper(String browserStackUser, String browserStackKey, int waitTimeoutInSeconds) {
+        Validate.notBlank(browserStackUser, "BrowserStack user must be provided, set the BROWSER_STACK_USER environment variable");
+        Validate.notBlank(browserStackKey, "BrowserStack key must be provided, set the BROWSER_STACK_KEY environment variable");
+        log.info("Using login: {} {}", browserStackUser, browserStackKey);
```

Comment:
Should be logging credentials.

Reply:
Yes, I'll remove in the next PR where I've fixed this and added logback-classic to the classpath so the logging actually works. The implication there is that this doesn't get logged at the moment so I'm happy to wait for the next PR.